_drafts/feature-tours.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: post
+title: "Feature Tours"
+description: TODO
+tags:
+- testing
+- ruby
+- web
+---
+
+You write tests that serve many different purposes. On one end of the spectrum, there are "unit tests" which isolate small units of behavior in your code to drive out a well-factored design. At the other extreme there are "feature" or "acceptance" tests that allow you to automate the use of a system as a user of it. These tests are great for specifying the system behavior of a _feature_. However, you sometimes still choose to isolate each use case of a feature in its own example. That's where _feature **tours**_ come in.
+
```

Comment:
This is interesting!  I've never heard the term "tour" used before.  Would you suggest that the way to determine when you need to create a new "tour" as opposed to adding to an existing one is when you need different setup or a different context for the data involved?

Reply:
Great point! Needing different context or setup would certainly push you towards separate examples. That said, the ideal outlined here would be to avoid contextual setup altogether when possible (warning: potential pipedream).